src/component/category-form/category-form.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import autoBind from '../../utils/';
+
+// this is the UI state. Everything but the store is the UI state from now on.
+const defaultState = {
+  title: '',
+};
+
+class CategoryForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = this.props.category || defaultState;
+    autoBind.call(this, CategoryForm);
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.onComplete(this.state);
+  }
+
+  render() {
+    const buttonText = this.props.category ? 'Update' : 'Create';
+    return (
+      <form 
+        onSubmit={this.handleSubmit}
+        className='category-form'>
+        <input
+          type='text'
+          name='name'  
```

Comment:
Change `name='name'` to `name='title` --> this is related to the property names in your state.